jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -911,7 +911,73 @@ ecma_date_insert_num_with_sep (ecma_string_t **str_p, /**< input/output string *
 } /* ecma_date_insert_num_with_sep */
 
 /**
- * Common function to create a time zone specific string.
+ * Common function to create a time zone specific string from a numeric value.
+ *
+ * Used by:
+ *        - The Date routine.
+ *        - The ecma_date_object_to_string helper routine.
+ *
+ * @return completion value
+ *         Returned value must be freed with ecma_free_completion_value.
+ */
+ecma_completion_value_t
+ecma_date_value_to_string (ecma_number_t *datetime_num_p, /**<datetime */
```

Comment:
Could you, please, change this argument to be number value, instead of pointer to value?